### ISSUE TITLE
Support Windows

### ISSUE THREAD

**Issue by rui314** [bluewhalesystems/sold#8]
We want to support Windows by sold 3.0.

**Comment by S-o-T** [comment]
Will windows version be a drop-in replacement for msvc link.exe? Will it support an incremental linking?

**Comment by rui314** [comment]
Yes and no. It'll be a drop-in replacement, but it won't support incremental linking. We'll instead make the regular linking extremely fast so that users don't even think about incremental linking.

**Comment by PoetaKodu** [comment]
You're an absolute chad bro, I hope your business succeeds.

**Comment by Sixmorphugus** [comment]
Definitely will be buying a licence when Windows is supported. Fantastic tool.

**Comment by thewh1teagle** [comment]
Hi, I'm really interested in adding Windows x86_64 support and would be eager to contribute if needed. Could you share the current status of Windows support and what steps would be required to implement it?

As a Rust developer, I've noticed that incremental compile times can take up to 10 seconds for some projects, which impacts the development experience. I'm particularly interested in enabling a seamless "hot reload" workflow for Rust development, and I believe this linker could play a key role in achieving that.
